core/domain/draft_upgrade_services.py

```diff
@@ -167,6 +166,31 @@ def extract_html_from_draft_change_list(draft_change_list):
 class DraftUpgradeUtil(python_utils.OBJECT):
     """Wrapper class that contains util functions to upgrade drafts."""
 
+    @classmethod
+    def _convert_states_v35_dict_to_v36_dict(cls, draft_change_list):
+        """Converts draft change list from version 35 to 36.
+
+        Args:
+            draft_change_list: list(ExplorationChange). The list of
+                ExplorationChange domain objects to upgrade.
+
+        Returns:
+            list(ExplorationChange). The converted draft_change_list.
+
+        Raises:
+            Exception. Conversion cannot be completed.
+        """
+        for change in draft_change_list:
+            if (change.property_name ==
+                    exp_domain.STATE_PROPERTY_INTERACTION_CUST_ARGS):
+                # Converting the customization arguments depends on getting an
+                # exploration state of v34. Since we do not yet support passing
```

Comment:
Maybe explain why that's the case (why it "depends on getting an exploration state of v34").